jbpm-workitem-itests/src/test/resources/kjars/kafka-process/src/main/java/com/myspace/kafka_process/BoxSerializer.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.myspace.kafka_process;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.Map;
+import org.apache.kafka.common.serialization.Serializer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.math.BigInteger;
+import java.util.List;
+import java.util.ArrayList;
+
+public class BoxSerializer implements Serializer<Box> {
```

Comment:
I noticed that serializer is in the kjar, but deserializer is outside of it. Is that on purpose?

Reply:
Yes, serializer is needed for publishing the message, meanwhile, deserializer is part of the assertion of the tests (not really needed in the producer side)

Reply:
I see, thanks.